pkgs/applications/networking/p2p/transmission/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, callPackage }:
+
+let
+  self = rec {
+    transmission_4 =
```

Comment:
How about defining `meta` in this file and passing it to `4-darwin.nix` and `4-linux.nix` to avoid duplication?

Reply:
i was thinking more of that if a linux maintainer changes the version and hash for linux, then he does not have a way to verify that the osx build still works. so my thinking wars that they are individually meta'd. Also because stuff like non-redist libaries like apple emoji's might be packed into it.